### ISSUE TITLE
Wrongly oriented 3D Keypoints and SMPL

### ISSUE THREAD

**Issue by YoussefFathi** [zju3dv/EasyMocap#160]
The visualization of the 3D keypoints for a multihuman dataset is producing keypoints below the z=0 as shown in the image below. This is due to having -ve values in the z-coordinate of the 3D keypoints. Is this an expected behaviour or does this mean there is an error in the predictions of the 3D keypoints? If there is an error, what could be the reason?

![image](https://user-images.githubusercontent.com/31455775/172338768-ffeacbac-58d5-47f2-a541-d281dadede26.png)

**Comment by YoussefFathi** [comment]
The dataset is already pre-calibrated in advance, so I have no control over the calibration parameters. I guess it was generated while assuming that the ground is not at z=0.

**Comment by chingswy** [comment]
Yes, you can modify the plane position for better visualization.
